DRAFT_RELEASE_NOTES.md

```diff
@@ -75,3 +75,5 @@ To start your aiSSEMBLE upgrade, update your project's pom.xml to use the 1.9.0
 1. Repeat the previous step until all manual actions are resolved
 
 # What's Changed
+- MLFlow Chart version upgraded from `0.2.1` to `1.4.22`. 
+- `aissemble-mlflow` Docker image using MLFlow 2.3.1 was replaced with `bitnami/mlflow:2.15.1-debian-12-r0`  Docker image using MLFlow 2.15.1.
```

Comment:
A: Add new migration to Automated Migrations table under the how to upgrade section.